apps/web-giddh/src/app/vouchers/list/list.component.ts

```
@@ -508,6 +503,12 @@ export class VoucherListComponent implements OnInit, OnDestroy {
                         this.settingForm.get('purchaseBillSettings.sendThroughGmail')?.enable();
                     }
                 });
+                if (!this.activeCompany?.subscription?.planDetails?.restrictedModules.hasOwnProperty(
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)